code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -582,3 +587,123 @@
 /obj/item/clothing/head/helmet/changeling/Initialize(mapload)
 	. = ..()
 	ADD_TRAIT(src, TRAIT_NODROP, CHANGELING_TRAIT)
+
+/datum/action/changeling/suit/hive_head
+	name = "Hive Head"
+	desc = "We coat our head in a waxy outing coating similar to a bee hive which can be used to manufacture bees to attack our enemies. Costs 15 chemicals."
+	helptext = "While the hive head does not provide much in the ways of armor, it does allow the user to send bees out to attack targets. Reagents can poured inside the hive to cause all bees released to inject said reagents."
+	button_icon_state = "hive_head"
+	chemical_cost = 15
+	dna_cost = 2
+	req_human = FALSE
```

Comment:
Double check how this combos with monkey ventcrawling, don't allow someone in pipes to spawn bees or legion skulls "from thin air" while in a pipe.